AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeView.swift

```diff
@@ -217,9 +247,15 @@ struct ChooseSessionTypeView: View {
                     mobileSessionButton
                 }
                 Spacer()
-                if featureFlagsViewModel.enabledFeatures.contains(.sdCardSync) {
-                    orLabel
-                    sdSyncButton
+                orLabel
```

Comment:
This way the or label will be present even if both of the below feature flags are turned off and there are no button. I think it shouldn't be like that, what do you think?

Reply:
Yeah, could look ugly. Will change